$OpenBSD$

XXX:
OpenBSD doesn't define EPROTO (yet), so pick the closest thing.

--- gdnsd/dnsio_tcp.c.orig	Sat Jul 26 02:37:00 2014
+++ gdnsd/dnsio_tcp.c	Sat Jul 26 02:40:00 2014
@@ -246,7 +246,7 @@ static void accept_handler(struct ev_loop* loop, ev_io
             case ENONET:
 #endif
             case ENETDOWN:
-            case EPROTO:
+            case EOPNOTSUPP:
             case EHOSTDOWN:
             case EHOSTUNREACH:
             case ENETUNREACH:
